api/app/signals/apps/feedback/serializers.py

```diff
@@ -64,4 +63,7 @@ def update(self, instance, validated_data):
                 }
                 Signal.actions.update_status(payload, signal)
 
+        tasks.send_system_mail.delay(
```

Comment:
When triggering the system mail for feedbakc received you can get the last received feedback in the ction additional context. No need to pass it here